data-prepper-plugins/rds-source/src/test/java/org/opensearch/dataprepper/plugins/source/rds/export/ExportTaskManagerTest.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.plugins.source.rds.export;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import software.amazon.awssdk.services.rds.RdsClient;
+import software.amazon.awssdk.services.rds.model.DescribeExportTasksRequest;
+import software.amazon.awssdk.services.rds.model.DescribeExportTasksResponse;
+import software.amazon.awssdk.services.rds.model.ExportTask;
+import software.amazon.awssdk.services.rds.model.StartExportTaskRequest;
+
+import java.util.List;
+import java.util.UUID;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+
+@ExtendWith(MockitoExtension.class)
+class ExportTaskManagerTest {
+
+    @Mock
+    private RdsClient rdsClient;
+
+    private ExportTaskManager exportTaskManager;
+
+    @BeforeEach
+    void setUp() {
+        exportTaskManager = createObjectUnderTest();
+    }
+
+    @Test
+    void test_start_export_task_without_specifying_tables() {
+        final String snapshotArn = "arn:aws:rds:us-east-1:123456789012:snapshot:snapshot-0b5ae174";
```

Comment:
Please generate random values for these (or a combination of random and non-random for ARNs).

Reply:
Updated to use random values whenever possible.